docs/02-Install.md

```diff
@@ -1,13 +1,111 @@
 # 2.0 - Install
 
-## 2.1 Dependencies
+## 2.1 Dependencies and installation
 
-## 2.2 Setup
+Dependencies and installation information are documented on the LORIS-MRI
+  [README.md](../README.md) file.
 
-## 2.3 Configuration
 
-## 2.4 Pipeline flow
+## 2.2 Configuration
 
-## 2.5 Common errors
+**NEED TO TALK ABOUT THE CREATION OF NIFTI OPTIONS AS IT IS REFERENCED IN THE
+  PIPELINE FLOW DOCUMENTATION.**
 
-[FAQ](AppendixA-FAQ.md)
+## 2.3 Pipeline flow
+
+By default the pipeline is designed for **raw DICOM MRI data**, collected by a
+  longitudinally-organized multisite study with a defined data acquisition
+  protocol. With modifications and further customization, it can handle any
+  modality of imaging data.
+
+Default images insertion into LORIS follows the following steps:
+
+1. Upload of a zip of the DICOM study via the **Imaging Uploader** module (or
+     DICOM study transfer directly on the server hosting LORIS).
+2. DICOM insertion into the `tarchive` tables in order to be able to see the
+     information stored in the DICOMs via the **DICOM Archive** module.
+3. Conversion of the DICOMs into MINC files for protocol validation and
+     insertion into the LORIS database. By default it will also create NIfTI
+     images from the MINC files and insert them into LORIS. See the
+     [Configuration](#2.2-configuration) section above if you want to disable
+     this option.
+     
+    a. If a scan matches one of the protocol defined in the `mri_protocol`
+         table, then the MINC image will be stored into the `files` tables and
+         can be visualized in the **Imaging Browser** module and
+         **BrainBrowser**.
+
+    b. If a scan does not match any of the protocol defined in the
+         `mri_protocol` table, then the MINC image of this scan will be stored
+         into the `mri_protocol_violated_scans` table and can be seen via the
+         **MRI Violated Scans** module.
+
+
+The graph below shows the different modules mentioned above with the
+  representation of the group of tables described in more details in the
+  [Technical Infrastructure](03-TechnicalInfrastructure.md) section. In
+  addition, the graph shows the name of the main script that is used to insert
+  the images into the LORIS database. More details about those scripts can be
+  found in the section [4 Scripts](04-Scripts.md).
+
+![pipeline_flow](images/overall_flow.png)
+
+## 2.4 Post-installation checks
```

Comment:
now that I am reading it as a whole; I feel this section comes out abruptly out of nowhere. What do you think if we put it inside section 2.1 as sections 2.1.1 and 2.1.2?